docs/overview.md

```diff
@@ -1,27 +1,41 @@
 # Overview
 
-## What is jrnl?
+`jrnl` is a simple but powerful plain text journal application for the command
+line.
 
-`jrnl` is a simple journal application for
-your command line. Journals are stored as human readable plain text
-files - you can put them into a Dropbox folder for instant syncing and
-you can be assured that your journal will still be readable in 2050,
-when all your fancy iPad journal applications will long be forgotten.
+Your imagination may be limitless, but your memory probably isn't. `jrnl` lets
+you write something down, and then it gets out of the way.
 
-Optionally, your journal can be encrypted using the [256-bit
-AES](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard).
+## how it works
 
-## Why keep a journal?
+New entries are created on the command line:
 
-Journals aren't just for people who have too much
-time on their summer vacation. A journal helps you to keep track of the
-things you get done and how you did them. Your imagination may be
-limitless, but your memory isn't.
+``` sh
+jrnl today at 8am: I arrived at work to find a birthday present on my desk. My colleagues are the best!
+```
 
-For personal use, make it a good habit to write at least 20 words a day.
-Just to reflect what made this day special, why you haven't wasted it.
+Support for external editors is included.
 
-For professional use, consider a text-based journal to be the perfect
-complement to your GTD todo list - a documentation of what and how
-you've done it. Or use it as a quick way to keep a change log. Or use it
-to keep a lab book.
+### one file type to rule them all...
```

Comment:
As mentioned above, the titles here should all be in title case.

Reply:
- Can you clarify what you mean about the `_dog_` thing? Delete it entirely, maybe? It was intended to emphasize ease of use, but it is kind of superfluous. Might be better on the advanced usage page.

- planning on changing titles to title case, and ditching the LOTR reference (unless people like it)